### PR TITLE
Fixed White Borders on pages and changed title from 'React App' to 'Hamlet'

### PR DESCRIPTION
**Pull request by swolbean**
Page borders updated:

- Home
- Account/Counter
- Checklist

image
image
image

### CODE REVIEW

**Comment by JasmineBautista** [comment]
nice, holding off approval until we get other PRs merged

**Review by JasmineBautista — APPROVED** [review]
dont merge till the other ones are

**Comment by stephenb-coding** [comment]
> dont merge till the other ones are

Good call

**Pull request merged**
